_scripts/app1-10.R

```diff
@@ -0,0 +1,38 @@
+{
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+  # Part A ----
+  
+  ## Prompt user to enter the age and location of a fish
+  
+  fishAge <- as.numeric(readline("What was the age of the fish? "));
+  fishLoc <- readline("Where was the fish caught (north or south)? ");
+  
+  # Part B ----
+
+  if((fishLoc == "north") && # != "south"
+     (fishAge >= 3 && fishAge <= 5)) # between ages 3 and 5
+  {
+    cat("Category I\n");
+  }
+  else if((fishLoc == "south") && # != "north"
+          (fishAge >= 2 && fishAge <= 6)) # between ages 2 and 6
+  {
+    cat("Category II\n");
+  }
+  else if(fishAge > 5) # age > 5 && port == "north" || "south"
```

Comment:
If I type in "west" as the location (or "east" or "Timbuktu") and age=7 then this codeblock will execute.